packages/devextreme/js/ui/overlay.d.ts

```diff
@@ -68,9 +61,9 @@ export interface dxOverlayOptions<TComponent> extends WidgetOptions<TComponent>
     deferRendering?: boolean;
     /**
      * @docid
-     * @deprecated dxOverlayOptions.wrapperAttr
+     * @deprecated
```

Comment:
don't we need to remove `deprecated` here?

Reply:
we discussed it in apiReview channel and dicided to keep @deprecated tag